### ISSUE TITLE
error == validation_data must be of tupple type

### ISSUE THREAD

**Issue by i-am-manish** [llSourcell/how_to_make_an_image_classifier#2]
import keras
from tensorflow.python.keras.models import Sequential
from tensorflow.python.keras.layers import Dense, Flatten, Conv2D, Dropout ,MaxPooling2D
import numpy as np

training_data=("C:/Users/Dell/Downloads/train")
validation_data=("C:/Users/Dell/Downloads/valid")

model=Sequential()
model.add(Conv2D(32,kernel_size=3,input_shape=(150,150,3),activation='relu'))#need to input
model.add(MaxPooling2D(pool_size=(2,2)))
model.add(Conv2D(32,kernel_size=3,activation='relu'))
model.add(MaxPooling2D(pool_size=(2,2)))
model.add(Conv2D(64,kernel_size=3,activation='relu'))
model.add(MaxPooling2D(pool_size=(2,2)))
model.add(Flatten()) #flats
model.add(Dense((64),activation='relu')) #FC layers
model.add(Dropout(0.5))
model.add(Dense((1),activation='sigmoid'))#1 probabilistic output

model.compile(loss='binary_crossentropy',optimizer='sgd',metrics=['accuracy'])

model.fit_generator(
        training_data,
        steps_per_epoch=2048,
        epochs=5,
        validation_data=validation_data,
        validation_steps=832)
